src/shared/metadata/selectors.js

```diff
@@ -398,10 +412,14 @@ export const getComputedCategoryOptionsByCatComboId = createCachedSelector(
 export const getSortedCoCsByCatComboId = createCachedSelector(
     (metadata) => metadata,
```

Comment:
While we're updating this code I think it's nice to have a comment as to why this is necessary. Since it seems like you would have access to the `categoryOptionCombos` directly from the `categoryCombo`.

```
/* The `categoryOptionCombos` of a `categoryCombo` is a Set, and therefore unordered. To be able to render the cocs (eg. an entry field) in the correct order (same order as the table-header), we need to compute the categoryOptionCombos client side. This is basically a `cartesian-product` of all the `categoryOptions` in all `categories` in the `categoryCombo. */
```

Reply:
I added the comment